src/ray/protobuf/export_api/export_train_state.proto

```diff
@@ -143,4 +150,5 @@ message ExportTrainRunAttemptEventData {
   repeated TrainResources resources = 8;
   // List of Train workers participating in this attempt, sorted by global ranks
   repeated TrainWorker workers = 9;
+  // WARNING: 10 is reserved for ray_train_version
```

Comment:
Is this just for code formatting instead of putting them all in ascending order?

Reply:
Yeah, though I guess it doesn't really make a difference. I can move it down